frontend/src/pages/TimeEstimationPage.tsx

```diff
@@ -110,12 +112,43 @@ export const TimeEstimationPage = () => {
     setCalculatedDaysPerWork(milestoneDuration / work);
   }, [selectedMilestoneId, milestoneDuration, roadmap, roadmapsVersions]);
 
+  const handleTooltipModal = () => {
+    dispatch(
+      modalsActions.showModal({
+        modalType: ModalTypes.INFO_MODAL,
+        modalProps: {
+          header: t('Estimate milestone durations'),
+          content: {
+            subHeader: t(
+              'Compare milestone durations with different estimates.',
+            ),
+            columns: [
+              {
+                header: t('Realization'),
+                text: t('system calculated average values over time.'),
+              },
+              {
+                header: t('Comparison'),
+                text: t('compare to another milestone'),
+              },
+              {
+                header: t('Custom'),
+                text: t(
+                  'estimate with a custom value for work amount in working days.',
+                ),
+              },
+            ],
+          },
+        },
```

Comment:
I couldn't figure out how to pass react components as params to the modal so I had to do this little workaround where I pass the data as an object with the desired fields and handle it in the modal. This approach should work well with the other infomodals that follow the same design.

Reply:
Yeah, the modal payload must be serialisable to json, so components are not possible as params.

The solution here seems good 👍